### ISSUE TITLE
Feature: Add Settings and/or preferences menu

### ISSUE THREAD

**Issue by RCFromCLE** [RCFromCLE/Solseek#12]
Currently, the application is devoid of any settings customization. As this is a desktop application, there should be a few accessibility options, theme options, currency view options, etc.

Please add a settings menu and any needed settings mentioned and not mentioned.
